src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/ResourceLocator.js

```diff
@@ -122,31 +133,36 @@ class ResourceLocator extends React.Component<FieldTypeProps<?string>> {
         }
 
         return (
-            <div className={resourceLocatorStyles.resourceLocatorContainer}>
-                <div className={resourceLocatorStyles.resourceLocator}>
-                    <ResourceLocatorComponent
-                        disabled={!!disabled}
-                        id={dataPath}
-                        mode={this.mode}
-                        onBlur={this.handleBlur}
-                        onChange={onChange}
-                        value={value}
+            <div>
```

Comment:
Wouldn't a `Fragment` do the trick as well? Or do you need a `div` without a `class` here for some reason?

Reply:
👍